style.css

```diff
@@ -1,336 +1,704 @@
+    width: 128px;
+    background: #36b37f;
+    border-radius: 6px 6px 6px 6px;
+    border: none;
+    display: flex;
+    color: #fff;
+    font-size: 19px;
+    margin-left: 100px;
+    padding-top: 14px;
+    padding-left: 13px;
+  }
+
+  /* ******************* section Two Above *********** */
+
+  .ab-btn-btn {
+    margin-right: 90px;
+    width: 160px;
+    margin-left: 0;
+    padding-left: 10px;
+    margin-bottom: 12px;
+  }
+
+  .about {
+    display: flex;
+    justify-content: space-evenly;
+    margin-top: 100px;
+  }
+
+  .about-li {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    margin-left: 25px;
+  }
+
+  .ab-head {
+    margin-top: 60px;
+    color: #344563;
+    font-size: 40px;
+  }
+
+  .side2 {
+    margin-left: 10px;
+  }
+
+  .side1 {
+    margin-right: 70px;
+  }
+
+  .ab-text {
+    font-size: 15px;
+    line-height: 28px;
+    text-align: center;
+    font-weight: 400;
+    color: #42526e;
+  }
+
+  .ab-lang {
+    background: #ebf0ee;
+    height: 352px;
+    margin-top: 50px;
+    margin-left: 25px;
+  }
+
+  .ab-img {
+    margin-top: 20px;
+  }
+
+  .ab-title {
+    margin-top: 10px;
+    padding-top: 0;
+    color: #344563;
+  }
+
+  .flex-tool {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .ab-li {
+    padding: 0;
+  }
+
+  .ab-btn {
+    background: #fff;
+    color: #36b37e;
+    display: flex;
+    padding-left: 15px;
+    padding-right: 10px;
+    padding-top: 10px;
+  }
+
+  /* ******************* section Three Above *********** */
+
+  #abt-contact {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .close {
+    margin-left: 25px;
+    margin-top: 50px;
+    background: #fff;
+    border: none;
+  }
+
+  .closure {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    text-align: center;
+    margin-right: 25px;
+  }
+
+  .ab-form {
+    margin-left: 25px;
+  }
+
+  input[type="text"] {
+    height: 3rem;
+    width: 20rem;
+    margin-bottom: 2rem;
+    border-radius: 6px 6px 6px 6px;
+    border: #cfd0d1 solid 1px;
+    padding-left: 1rem;
+    font-size: 1rem;
+  }
+
+  input[type="email"] {
+    height: 3rem;
+    width: 20rem;
+    margin-bottom: 2rem;
+    border-radius: 6px 6px 6px 6px;
+    border: #cfd0d1 solid 1px;
+    padding-left: 1rem;
+    font-size: 1rem;
+  }
+
+  textarea {
+    height: 8rem;
+    width: 20rem;
+    border-radius: 6px 6px 6px 6px;
+    border: #cfd0d1 solid 1px;
+    padding-left: 1rem;
+    font-size: 1rem;
+  }
+
+  input[type="submit"] {
+    padding-bottom: 1rem;
+    margin-top: 30px;
+  }
+
+  .send {
+    height: 50px;
+    width: 128px;
+    background: #36b37f;
+    border-radius: 6px 6px 6px 6px;
+    border: none;
+    display: flex;
+    color: #fff;
+    font-size: 19px;
+    padding-top: 14px;
+    padding-left: 13px;
+  }
+
+  .footer {
+    display: flex;
+    height: 10px;
+  }
```

Comment:
Kindly adjust the footer to match that of your chosen design template. The social icons are centered on the page and the top and bottom margins are smaller than shown on your design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/200076291-97404af5-8eaa-479e-9261-4684900fcfde.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/200076368-e62a9591-094f-4471-8f8d-69a210eb6ff2.png)